static/app/components/events/viewHierarchy/node.tsx

```diff
@@ -0,0 +1,93 @@
+import styled from '@emotion/styled';
+
+import {IconAdd, IconSubtract} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+
+type NodeProps = {
+  id: string;
+  isExpanded: boolean;
+  label: string;
+  onExpandClick: () => void;
+  collapsible?: boolean;
+  isSelected?: boolean;
+  onSelection?: () => void;
+};
+
+function Node({
+  label,
+  id,
+  onSelection,
+  isExpanded,
+  isSelected,
+  onExpandClick,
+  collapsible = false,
+}: NodeProps) {
+  return (
+    <NodeContents aria-labelledby={`${id}-title`}>
+      <IconWrapper
+        aria-controls={id}
+        aria-label={isExpanded ? t('Collapse') : t('Expand')}
+        aria-expanded={isExpanded}
+        isExpanded={isExpanded}
+        onClick={onExpandClick}
+        collapsible={collapsible}
+      >
+        {isExpanded ? (
+          <IconSubtract legacySize="9px" color="white" />
+        ) : (
+          <IconAdd legacySize="9px" color="white" />
```

Comment:
minor thing, i noticed `legacySize` is deprecated. are we using `legacySize` because the values for `size` aren't small enough?

Reply:
Yeah, when I originally wrote this I borrowed the expand/collapse buttons from the key-value component and at some point `size` became a legacy prop since we wanted to prefer `size="xs","sm", etc`

`xs` is still too large so I figured we'd use `legacySize` until there's a solution for the key-value component as well